rpm/del-fixtures-sign.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
```

Comment:
Please declare the script as being a bash script, not a sh script. Some systems will ignore the request and start a bash shell anyway, but that's not something to count on.